articles/app-service/app-service-web-tutorial-connect-msi.md

```diff
@@ -136,29 +136,60 @@ Type `Ctrl+F5` to run the app again. The same CRUD app in your browser is now co
 In Visual Studio, open the Package Manager Console and add the NuGet package [Microsoft.Azure.Services.AppAuthentication](https://www.nuget.org/packages/Microsoft.Azure.Services.AppAuthentication):
 
 ```powershell
-Install-Package Microsoft.Azure.Services.AppAuthentication -Version 1.4.0
+Install-Package Microsoft.Data.SqlClient -Version 2.1.2
+Install-Package Azure.Identity -Version 1.4.0
 ```
 
 In the [ASP.NET Core and SQL Database tutorial](tutorial-dotnetcore-sqldb-app.md), the `MyDbConnection` connection string isn't used at all because the local development environment uses a Sqlite database file, and the Azure production environment uses a connection string from App Service. With Active Directory authentication, you want both environments to use the same connection string. In *appsettings.json*, replace the value of the `MyDbConnection` connection string with:
 
 ```json
-"Server=tcp:<server-name>.database.windows.net,1433;Database=<database-name>;"
+"Server=tcp:<server-name>.database.windows.net;Authentication=Active Directory.Device Code Flow; Database=<database-name>;"
```

Comment:
My understanding here is that the value for the `Authentication` keyword is not important, but needs to match the one from our custom `SqlAuthenticationProvider`.

Is that correct?
If it is, then I'd suggest we either explain why we picked the "device code flow" authentication type.

Reply:
I agree. I have the same question about the type choice (and looks like there's a . in there that shouldn't be there according to the AAD documentation). And if it's not important, then why the need to specify at all?

Reply:
@mderriey @cephalin added a note that explains why AuthCode Flow was chosen here